scenarios/getting_started/go/getting_started.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/eclipse/paho.golang/paho"
+)
+
+type MqttConnectionSettings struct {
+	Hostname  string
+	TcpPort   int
+	UseTls    bool
+	CaFile    string
+	KeepAlive uint16
+	ClientId  string
+	Username  string
+	KeyFile   string
+}
+
+func main() {
+	// Load connection settings
+
+	cs := ConnectionSettings.loadConnectionSettings("../.env")
```

Comment:
we should not hardcode the path to the .env file, or we accept as an argument, or we try to load from current dir.